compiler/src/dmd/expression.h

```diff
@@ -50,6 +50,7 @@ namespace dmd
 {
     // in expressionsem.d
     Expression *expressionSemantic(Expression *e, Scope *sc);
+    void lowerNonArrayAggregate(StaticForeach *sfe, Scope *sc);
```

Comment:
you will need to add this to https://github.com/dlang/dmd/blob/master/compiler/src/dmd/frontend.h, 
see the CircleCI failure

```diff
diff --git a/home/circleci/dmd/compiler/src/dmd/frontend.h b/home/circleci/dmd/generated/linux/release/64/frontend.h
index 5f4924297b..82c1fe5c12 100644
--- a/home/circleci/dmd/compiler/src/dmd/frontend.h
+++ b/home/circleci/dmd/generated/linux/release/64/frontend.h
@@ -7550,6 +7550,8 @@ public:
     void visit(StaticForeachDeclaration* sfd) override;
 };
 
+extern void lowerNonArrayAggregate(StaticForeach* sfe, Scope* sc);
+
 class NrvoWalker final : public StatementRewriteWalker
 {
 public:
```